.swiftpm/xcode/xcshareddata/xcschemes/BrowserServicesKit-Package.xcscheme

```diff
@@ -413,6 +413,20 @@
                ReferencedContainer = "container:">
             </BuildableReference>
          </BuildActionEntry>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "YES"
+            buildForProfiling = "YES"
+            buildForArchiving = "YES"
+            buildForAnalyzing = "YES">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "CxxCrashHandler"
+               BuildableName = "CxxCrashHandler"
+               BlueprintName = "CxxCrashHandler"
+               ReferencedContainer = "container:">
+            </BuildableReference>
+         </BuildActionEntry>
```

Comment:
Is this part of the PR?

Reply:
My Xcode keeps bugging me about adding this. It's an update to the otherwise auto-generated scheme, so I reckon it doesn't hurt to include it.